src/translations.php

```diff
@@ -261,6 +261,14 @@
         "Present" => "Vandaag",
         "Excluding" => "Exclusief",
     ],
+    "ph_CB" => [
```

Comment:
Thanks for contributing the translation!

It looks like the code for this one should be [`ceb`](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?code_ID=77)

```suggestion
    "ceb" => [
```

You can commit the change to your branch to update the PR, also make sure when its changed its still in alphabetical order. Let me know if you have any questions. Thanks!